ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/FilterResourceDropdown.tsx

```diff
@@ -52,7 +52,7 @@ function FilterResourceDropdown({
             onSelect={onResourceSelect}
             selections={resource}
             isOpen={isOpen}
-            className="pf-u-w-25"
+            style={{ flex: 0 }}
```

Comment:
what's the difference between doing this and adding `pf-u-flex-none` as a class here?

Reply:
I think the main differences are:
1. `flex: 0` will expand to `flex: 0 1 0` while `pf-u-flex-none` would expand to `flex: 0 0 auto` (functionally the same in this case)
2. `pf-u-flex-none` is cleaner than the inline style
3. I didn't know PatternFly had `pf-u-flex-none` 🙂 

I'll update this before merging.